src/Mvc/src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs

```diff
@@ -162,28 +126,11 @@ public bool AllowCombiningAuthorizeFilters
         /// Gets or sets a value that determines if <see cref="HeaderModelBinder"/> should bind to types other than
         /// <see cref="string"/> or a collection of <see cref="string"/>. If set to <c>true</c>,
         /// <see cref="HeaderModelBinder"/> would bind to simple types (like <see cref="string"/>, <see cref="int"/>,
-        /// <see cref="Enum"/>, <see cref="bool"/> etc.) or a collection of simple types. The default value of the
-        /// property is <c>false</c>.
+        /// <see cref="Enum"/>, <see cref="bool"/> etc.) or a collection of simple types.
```

Comment:
It looks like stuff is still using compatibility switch as infrastructure (L121) - is your plan to remove that in a follow up?

Reply:
Yes, that's the plan